pom.xml

```diff
@@ -9,7 +9,7 @@
     <relativePath />
   </parent>
   <artifactId>aws-java-sdk</artifactId>
-  <version>1.11.120-SNAPSHOT</version>
+  <version>1.11.211-SNAPSHOT</version>
```

Comment:
why increment this version? shouldn't it be part of the maven release process?

Reply:
Doesn't really matter. This is mostly to get the right default value when doing the maven release later on.